### PR TITLE
Correct property name of offset

### PR DESCRIPTION
**Pull request by Jenselme**

### CODE REVIEW

File: src/services/print.js [hunk 1]
```diff
@@ -656,8 +656,11 @@ ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
       symbolizer.fontColor = goog.color.rgbArrayToHex(fillColorRgba);
     }
 
-    symbolizer.XOffset = textStyle.getOffsetX();
-    symbolizer.YOffset = textStyle.getOffsetY();
+    // Mapfish print allows offset only if labelAlign is defined.
+    if (goog.isDef(symbolizer.labelAlign)) {
+      symbolizer.labelXOffset = textStyle.getOffsetX();
+      symbolizer.labelYOffset = - textStyle.getOffsetY();
```
Comment (elemoine) [inline src/services/print.js]: You did not have the minus sign previously. Can you please confirm it's needed?

Comment (elemoine) [inline src/services/print.js]: Also, could you please remove the space character between the minus sign and `textStyle`?

**Comment by elemoine** [comment]
Thanks for the patch. I'll merge it when the comments are addressed.

**Comment by Jenselme** [comment]
PR updated:
- space between the minus sign and `textStyle` is removed
- add a comment to underline the necessity of the minus sign.

File: src/services/print.js [hunk 2]
```diff
@@ -656,8 +656,13 @@ ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
       symbolizer.fontColor = goog.color.rgbArrayToHex(fillColorRgba);
     }
 
-    symbolizer.XOffset = textStyle.getOffsetX();
-    symbolizer.YOffset = textStyle.getOffsetY();
+    // Mapfish print allows offset only if labelAlign is defined.
```
Comment (elemoine) [inline src/services/print.js]: Mapfish print -> MapFish Print

File: src/services/print.js [hunk 3]
```diff
@@ -656,8 +656,13 @@ ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
...
+    if (goog.isDef(symbolizer.labelAlign)) {
+      symbolizer.labelXOffset = textStyle.getOffsetX();
+      // Mapfish uses the opposite direction of OpenLayer for y axis, so the
```
Comment (elemoine) [inline src/services/print.js]: Mapfish -> MapFish Print

Comment (elemoine) [inline src/services/print.js]: OpenLayer -> OpenLayers

**Comment by Jenselme** [comment]
Projects name corrected.

**Pull request merged**